src/plugins/discover/public/application/main/hooks/use_adhoc_data_views.ts

```diff
@@ -48,8 +52,9 @@ export const useAdHocDataViews = ({
         const existing = prev.find((prevDataView) => prevDataView.id === dataView.id);
         return existing ? prev : [...prev, dataView];
       });
+      trackUiMetric?.(METRIC_TYPE.CLICK, ADHOC_DATA_VIEW_RENDER_EVENT);
```

Comment:
Maybe it should be rather `METRIC_TYPE.LOAD` since it fires also on initial page render? Same for a saved search. Wdyt? cc @kertal

Reply:
this is correct, but METRIC_TYPE.CLICK is not wrong, so I'd suggest to use `METRIC_TYPE.COUNT`, like Lens is doing, this is more neutral, and covers both cases